site/src/pages/community/developer-guide.mdx

```diff
@@ -2,7 +2,7 @@
 
 ## Build requirements
 
-- [OpenJDK 11-19](https://adoptium.net/)
+- [OpenJDK 17](https://adoptium.net/)
```

Comment:
@jrhee17 has changed this line in the PR. 😄 
https://github.com/line/armeria/pull/4643/files#diff-7073dd8ed130d2631ff065c9a9e8c98db571d01627480773a2551746c0d1f7ddR5

> Should we say 'OpenJDK 17 or its derivatives' instead of just OpenJDK by the way?

Sounds good! 😄

Reply:
I'm fine to rollback my changes on this in my PR 😄